### DISCUSSION TITLE
Questions about HALFpipe

### DISCUSSION THREAD

**Discussion by Huaiyu-Liu** [HALFpipe/HALFpipe#734] [Q&A] [answered]
Dear HALFpipe community,

I am a novice to HALFpipe and therefore would like to to ask two questions:
1. I noticed that the latest fmriprep version is 24.00, hence I wonder do HALFpipe also uses this version?
2. I am a windows user, I wonder is there any compatibility issues for windows still?

**Comment by lalalavi** [comment, marked as answer]
Dear Huaiyu,

Welcome to HALFpipe :) In our latest release, we are using fmriprep version `20.2.7`.  Later this year, probably after the summer, we will upgrade fmriprep to the latest stable version available. If you have more doubts about which version of the dependencies we are using for other packages, you can see them all in the [requirements.txt](https://github.com/HALFpipe/HALFpipe/blob/main/requirements.txt) file.

You should also be able to run HALFpipe on your Windows computer. Some people do it via Docker Desktop, or you can also opt to run it via WSL (Windows Subsystem for Linux).

If you have any other question, let me know!
